src/saml2/entity.py

```diff
@@ -203,6 +203,40 @@ def __init__(self, entity_type, config=None, config_file="",
 
         self.msg_cb = msg_cb
 
+    def reload_metadata(self, metadata_conf):
+        """
+        Reload metadata configuration.
+
+	Load a new metadata configuration as defined by metadata_conf (by
```

Comment:
```suggestion
        Load a new metadata configuration as defined by metadata_conf (by
```

Reply:
Ooops, a Tab did sneak through - thanks for catching that!